db/products.ts

```diff
@@ -0,0 +1,47 @@
+/** ************************************************************
+ * THIS FILE IS FOR FUNCTIONALITY DEMONSTRATION PURPOSES ONLY
+ * NO NEED TO ASSESS THIS FILE OR ITS CONTENTS
+ * ********************************************************** */
+
+export const PRODUCTS = [
+  {
+    id: 1,
+    title: "Product 1",
+    price: 100,
+    stock: 10,
+    img: "https://picsum.photos/200/300",
+  },
+  {
+    id: 2,
+    title: "Product 2",
+    price: 200,
+    stock: 20,
+    img: "https://picsum.photos/200/200",
+  },
+  {
+    id: 3,
+    title: "Product 3",
+    price: 300,
+    stock: 30,
+    img: "https://picsum.photos/300/200",
+  },
+  {
+    id: 4,
+    title: "Product 4",
+    price: 400,
+    stock: 40,
+    img: "https://picsum.photos/300/300",
+  },
+  {
+    id: 5,
+    title: "Product 5",
+    price: 500,
+    stock: 50,
+    img: "https://picsum.photos/400/300",
+  },
+];
```

Comment:
For testing purpose I would maybe consider getting rid of just one stock and ask the potential candidate about formatting data (as I've seen yesterday during short meeting with @VED-martinwheeler we do that).
Better yet the typescript would complain a bit if we were to add a product later on in a task.